fastly/block_fastly_service_logging_gcs.go

```diff
@@ -38,6 +38,11 @@ func (h *GCSLoggingServiceAttributeHandler) GetSchema() *schema.Schema {
 			DefaultFunc: schema.EnvDefaultFunc("FASTLY_GCS_ACCOUNT_NAME", ""),
 			Description: "The google account name used to obtain temporary credentials (default none). You may optionally provide this via an environment variable, `FASTLY_GCS_ACCOUNT_NAME`.",
 		},
+		"project_id": {
```

Comment:
This new field needs to be placed in alphabetical order (so just before the "secret_key" field). This is what the CI 'lint' job is failing on currently.

Reply:
Thank you, fixed in e2e4b46f2acc808712cb6bb700c99457294d420d.